package.json

```diff
@@ -12,8 +12,9 @@
     "image:build": "./scripts/build-image.sh",
     "image:push": "./scripts/push-image.sh",
     "clean": "yarn rimraf dist",
-    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix": "yarn eslint . --ext .js,.jsx,.ts,.tsx --fix ",
+    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx && yarn prettier ./src --check",
```

Comment:
Won't `yarn prettier ./src --check` also run on all the JS/TS? With the eslint plugin, that would run prettier on those files twice.

Reply:
You're right, that was a test that I forgot to change. Fixed